torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -281,12 +283,13 @@ struct CudaGraphFuser {
       mergeFusionGroups(group, producer->node());
       return group;
     }
-    AT_ASSERT(producer->node()->outputs().size() == 1);
+    // AT_ASSERT(producer->node()->outputs().size() == 1);
```

Comment:
why is this commented out?

Reply:
`torch.native_layer_norm` and `torch.native_layer_norm_backward` generate 3 outputs apiece. Hence, this assertion will fail.